<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wolframe Qt Client</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="themeAuthor" stdset="0">
   <string>Andreas Baumann</string>
  </property>
  <property name="themeBaseStyle" stdset="0">
   <string>plastique</string>
  </property>
  <widget class="QMdiArea" name="centralWidget">
   <property name="viewMode">
    <enum>QMdiArea::SubWindowView</enum>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionRestart"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="actionIndex"/>
    <addaction name="actionContextHelp"/>
    <addaction name="separator"/>
    <addaction name="actionTechnical_Support"/>
    <addaction name="actionReport_Bug"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_for_Updates"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
    <addaction name="separator"/>
    <addaction name="actionDebugTerminal"/>
   </widget>
   <widget class="QMenu" name="menuLanguages">
    <property name="title">
     <string>&amp;Language</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="action_ReloadWindow"/>
    <addaction name="actionOpenForm"/>
    <addaction name="actionOpenFormNewWindow"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
    <addaction name="separator"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Window"/>
   <addaction name="menuLanguages"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>open.png</normaloff>open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="qtclient.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About Wolframe Client</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionDebugTerminal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Debug Terminal</string>
   </property>
   <property name="iconText">
    <string>Debug Terminal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>&amp;Language</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="qtclient.qrc">
     <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionManageStorage">
   <property name="text">
    <string>&amp;Manage Storage</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>&amp;Restart</string>
   </property>
  </action>
  <action name="actionOpenForm">
   <property name="text">
    <string>&amp;Open form..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpenFormNewWindow">
   <property name="text">
    <string>Open form in &amp;new window..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Contents</string>
   </property>
  </action>
  <action name="actionIndex">
   <property name="text">
    <string>Index</string>
   </property>
  </action>
  <action name="actionContextHelp">
   <property name="text">
    <string>Context Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionTechnical_Support">
   <property name="text">
    <string>Technical Support</string>
   </property>
  </action>
  <action name="actionReport_Bug">
   <property name="text">
    <string>Report Bug...</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates...</string>
   </property>
  </action>
  <action name="action_ReloadWindow">
   <property name="icon">
    <iconset resource="qtclient.qrc">
     <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reload Window</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>Close &amp;all</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Tile</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Cascade</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qtclient.qrc">
     <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
   </property>
   <property name="text">
    <string>Ne&amp;xt</string>
   </property>
   <property name="toolTip">
    <string>Move the focus to the next window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qtclient.qrc">
     <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>Pre&amp;vious</string>
   </property>
   <property name="toolTip">
    <string>Move the focus to the previous window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Backtab</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qtclient.qrc"/>
 </resources>
 <connections/>
</ui>
